web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1393,6 +1393,9 @@ fixmystreet.display = {
                 $('html').addClass('only-map');
                 $('#mob_sub_map_links').removeClass('map_complete');
                 $('#mob_ok').text(translation_strings.ok);
+                if (fixmystreet.duplicates) {
+                    fixmystreet.duplicates.hide();
```

Comment:
I think this does make sense at present, especially given there's no way to then dismiss the duplicates without going back down anyway. Perhaps on mobile it'd be better if duplicates weren't shown on the map at all ever, and instead when you clicked "Read more" under any of them, the expansion included a little inline map there? That would keep them together then at least.

Reply:
💯 to the inline map: #2668.